### ISSUE TITLE
x11docker on AWS

### ISSUE THREAD

**Issue by fabianomenegidio** [mviereck/x11docker#39]
Could I run x11docker on a ubuntu server installation with docker and dependencies installed? How could I create more than one instance on different VNC ports?

**Comment by mviereck** [comment]
>Could I run x11docker on a ubuntu server installation with docker and dependencies installed? 

Yes, of course.

> How could I create more than one instance on different VNC ports?

I have updated README.md to make this a bit more clear:

```
read Xenv < <(x11docker --xdummy --showenv x11docker/lxde)
echo $Xenv
env $Xenv x11vnc -noshm -forever -localhost -rfbport 5910
```
 In another terminal, start VNC viewer with `vncviewer localhost:5910`.

For multiple instances of VNC use different port numbers for each instance.
This example provides a quite simple setup on localhost only and without a password; it is enough to get a working example.

`x11vnc -help` shows a big bunch of options I am not familar with. It would be up to you to find out which options in detail you would need.

-------------
Depending on what you want to achieve you may be served better with an xpra SSH setup: https://github.com/mviereck/x11docker#ssh-with-xpra
xpra provides better performance and the connection is secured with SSH. (Though, combining VNC with SSH is somehow possible, too).

Also it is possible to run xpra within a container without using x11docker at all. Network setup with x11docker is possible, but it is more designed for local use cases, and it does not have that much advantages in a network setup. 

Though, x11docker may be quite useful if you want to use a GPU on the server.

**Comment by fabianomenegidio** [comment]
Tnx.

**Comment by mviereck** [comment]
A quite easy attempt is using `ssh -X`. 
```
# single application, no dependencies at all on server beside docker and x11docker
ssh -X user@server
x11docker  x11docker/xfce thunar
```

```
# VNC like setup, needs  Xephyr, docker and x11docker on server
ssh -X user@server
x11docker  --desktop x11docker/xfce
```
A one-liner like `ssh -X user@server -- x11docker  x11docker/xfce thunar` currently fails, I have to investigate that.

**Comment by mviereck** [comment]
> A one-liner like ssh -X user@server -- x11docker x11docker/xfce thunar currently fails, I have to investigate that.

The ssh bug is fixed now in release v4.2.0.
